src/lib/track.js

```diff
@@ -10,6 +11,16 @@ import type {
 
 export const track = <T>(config : Config, trackingType : TrackingType, trackingData : T) => {
     const encodeData = data => encodeURIComponent(btoa(JSON.stringify(data)));
+    let cartId;
+
+    try {
+        // $FlowFixMe
+        cartId = getCartId().cartId;
+    } catch (err) {
```

Comment:
[nit] I assume the catch is here because getCartId() could return `null`, and `(null).cartId` would throw an error? Seems a little unintuitive, I might do `cartId = getCartId() ? getCartId().cartId : createNewCartId().cartId`

Reply:
Agreed. It isn't intuitive.

With the current implementation this _should_ never happen. Ideally 'track' should just read from the cartId in localstorage. If 'track' is getting called and it doesn't exist it's basically a programmer error. I wanted to surface that in some way.

Reply:
Done. Created a new method 'getOrCreateCartId' that explicitly get or sets.